uwblueprint/planet-read
# Delete story translations in StoryTranslationTable

## Notion ticket link



[Delete story translations on StoryTranslationTable](https://www.notion.so/uwblueprintexecs/Delete-story-translations-on-StoryTranslationTable-3c14db074c2941fc8cb1e2d8e10daaf2)



## Implementation description

- make soft delete call in story translations table
- added underline to links
- accidentally updated prettier version locally and applied the changes to the `package.json` as well -> it now seems to match vscode formatting complaints ? ? 



## Steps to test

1. Rebuild frontend (`docker-compose up --build`)
2. Login as Angela Merkel
3. Go to /admin page and try to delete a story translation
4. Verify that story translation does not appear in the table anymore

![delete translation](https://user-images.githubusercontent.com/32009013/139615557-1c02ac79-4078-4232-9101-07b2ebc1f457.gif)

underlined links:
![image](https://user-images.githubusercontent.com/32009013/139984531-9395e703-2d8c-4100-bd91-8bce6daf928e.png)




## What should reviewers focus on?
- `frontend/src/components/admin/StoryTranslationsTable.tsx`
- does it compile locally ? 

## Checklist
